python_ta/typecheck/type_store.py

```diff
@@ -35,3 +36,20 @@ def __init__(self):
                     concrete_arg_types.append(concrete_arg_type)
                 self.classes[class_def.name][function_def.name] = (Callable[concrete_arg_types, rtype], class_def.name)
                 self.functions[function_def.name].append(Callable[concrete_arg_types, rtype])
+
+    def lookup_function(self, operator, *args):
+        """Helper method to lookup a function type given the operator and types of arguments."""
+        function_type = None
+        if args:
+            funct_types_list = self.functions[operator]
+            found = False
+            for funct_type in funct_types_list:
+                if funct_type.__args__[:-1] == args:
+                    function_type = funct_type
```

Comment:
You can just return `funct_type` here directly.

Also, the more conventional abbreviation for "function" is "func".